### PR TITLE
New version: primecount-7.2

### PR DESCRIPTION
**Pull request by kimwalisch**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37101)

### CODE REVIEW

File: manifests/p/Primecount/Primecount/7.2/Primecount.Primecount.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+PackageIdentifier: Primecount.Primecount
+Publisher: Primecount
```
Comment (ItzLevvie) [inline manifests/p/Primecount/Primecount/7.2/Primecount.Primecount.yaml:2]:
```suggestion
Publisher: Kim Walisch
```

Comment (ItzLevvie) [reply manifests/p/Primecount/Primecount/7.2/Primecount.Primecount.yaml:2]: This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/p/Primecount/Primecount/7.2/Primecount.Primecount.yaml (hunk 1 above)
Comment (kimwalisch) [reply manifests/p/Primecount/Primecount/7.2/Primecount.Primecount.yaml]: I have fixed the Publisher: https://github.com/microsoft/winget-pkgs/pull/37101/commits/e9ad7ded207268afa0d91157d61308ec21018467

**Pull request merged**